sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/fullfidelity/ChangeFeedOperationType.java

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.changefeed.fullfidelity;
+
+import com.azure.cosmos.util.Beta;
+
+@Beta(value = Beta.SinceVersion.V4_28_0)
+public enum ChangeFeedOperationType {
```

Comment:
Same here. this is implementation. We don't need Beta annotations here. 
Also, are we planning to ship `ChangeFeedOperationType` and `ChangeFeedMetaData` as part of `handle()` API response?